packages/rpm/centos/mariadb-10.1-mroonga.spec.in

```diff
@@ -5,8 +5,12 @@
 %define mariadb_download_base_url_default https://downloads.mariadb.org/f
 %if %{_centos_ver} == 7
 %define mariadb_dist_default    el7.centos
+%define service_status() (systemctl status %1 > /dev/null)
```

Comment:
How about removing `> /dev/null` from macro?
It's common in both `systemctl` version and `/sbin/service` version.

Reply:
```shell
 if ! %{service_status mysqld} > /dev/null; then
```

Reply:
Thanks, unified to use `> /dev/null` style for consistency.

Reply:
Really?
The diff still has `> /dev/null` in macro.

Reply:
Thanks, still `> /dev/null` exists... this is not intended, so I've fixed.